MyXamarinApp.API/Repository/EmployeeRepository.cs

```diff
@@ -39,17 +46,30 @@ public async Task<EmployeeModel> AddEmployee(EmployeeModel employeeModel)
             var employee = _mapper.Map<Employee>(employeeModel);
             _context.Employees.Add(employee);
             await _context.SaveChangesAsync();
-            return employeeModel;
+            return _mapper.Map<EmployeeModel>(employee);
```

Comment:
is the guid on employee variable and the one saved in the database going to be same ?

Reply:
Yes